src/lib.rs

```diff
@@ -17,12 +17,12 @@
 
 #![doc(html_root_url = "https://hyperium.github.io/mime.rs")]
 #![cfg_attr(test, deny(warnings))]
-#![cfg_attr(all(features = "nightly", test), feature(test))]
+#![cfg_attr(all(feature = "nightly", test), feature(test))]
 
 #[macro_use]
 extern crate log;
 
-#[cfg(features = "nightly")]
+#[cfg(feature = "nightly")]
```

Comment:
can you change this to `#[cfg(all(feature = "nightly", test))]` so that it is still possible to run tests but not benchmarks on stable? (I may be remembering how this works wrong and it might work already)